### PR TITLE
#166980203 Setup react router version 4

### PR DESCRIPTION
**Pull request by wekesa931**
#### What does this PR do?
- Sets up react router version 4
#### Description of Task to be completed?
- install react-router-dom
- Create dummy routes to illustrate how the router works
#### How should this be manually tested?
* Pull the branch:
    `git pull origin ch-react-router-setup-166980203`
* Install dependencies:
    `yarn`
* Run local server:
    `yarn develop`
* Access the routes
```
1. Home - http://localhost:3000
2. Login - http://localhost:3000/login
```
#### Any background context you want to provide?
React router is a routing library built on top of the react which is used to create the routing in react apps.
#### What are the relevant pivotal tracker stories?
[#166980203](https://www.pivotaltracker.com/story/show/166980203)
#### Screenshots (if appropriate)
1. Home page
Screenshot 2019-07-02 at 18 35 23

2. Login page
Screenshot 2019-07-02 at 18 35 32

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import React from "react";
+import Routes from '../routes';
 
 function App() {
-  return <div>Hello World!</div>;
+  return (
+    <div>
```
[the remark this answers is not in the document]
Comment (wangonya) [reply src/components/App.js:6]: Add `"react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]` to your `rules` in the `.eslintrc` file to fix this.

**Review by wangonya — CHANGES_REQUESTED** [review]
Kindly fix the PR description in the `How should this be manually tested?` section to match your branch.

**Review by jmusila — COMMENTED** [review]
Maybe you could check on resolving the hound violations.

**Review by willeswa — COMMENTED** [review]
There seem to be lots of complaints from Hound. Are you planning to have them fixed?

**Review by wangonya — CHANGES_REQUESTED** [review]
You should add `Home` as a link so we can be able to navigate between `Login` and `Home`. At the moment, navigating to `Login` via the link works, but I have to manually edit the url to go back to `Home`.

**Review by wangonya — APPROVED** [review]

File: src/components/App.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (wekesa931) [reply src/components/App.js:6]: I have added to the rules

**Pull request merged**